engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -57,45 +37,31 @@ object MetadataServiceActor {
   case class MetadataLookupResponse(query: MetadataQuery, eventList: Seq[MetadataEvent]) extends MetadataServiceResponse
   case class MetadataServiceKeyLookupFailed(query: MetadataQuery, reason: Throwable) extends MetadataServiceResponse
 
-  def props(serviceConfig: Config, globalConfig: Config) = {
-    Props(MetadataServiceActor(serviceConfig, globalConfig))
+  def props(dataAccess: DataAccess, serviceConfig: Config, globalConfig: Config) = {
```

Comment:
TOL, is there no possible use case where we would want to actually have an in-memory MetadataService ?
Since they're swappable via the conf file anyway, could we have both an  `InMemoryMetadataActor` and a `PersistenceMetadataActor` living side by side ?